src/main/java/com/rtbhouse/kafka/workers/impl/consumer/ConsumerThread.java

```diff
@@ -62,7 +62,7 @@ public ConsumerThread(
         super("consumer-thread", config, metrics, workers);
 
         this.consumerPollTimeout = Duration.ofMillis(config.getLong(WorkersConfig.CONSUMER_POLL_TIMEOUT_MS));
-        this.consumerProcessingTimeout = Duration.ofMillis(config.getLong(WorkersConfig.CONSUMER_PROCESSING_TIMEOUT_MS));
+        this.consumerProcessingTimeout = config.getConsumerProcessingTimeout();
```

Comment:
It's nice! However, not consistent with calls next to it.

Reply:
Added getConsumerPollTimeout() method returning Duration.

Reply:
There is yet another one call below :)

Reply:
Done
I thought it relates only to methods returning Duration